.github/workflows/update-helm.yaml

```diff
@@ -0,0 +1,77 @@
+name: Helm update
+
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  verify-helm-changes:
+    name: Verify if AKO helm charts needs updates
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout AKO repo
+        uses: actions/checkout@v4
+
+      - name: Configure git
+        run: |
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+
+      - name: Checkout Helm Repo
+        run: |
+          git clone https://github.com/mongodb/helm-charts.git helm-charts-cloned
+
+      - name: Verify if CRDs were changed
+        id: crd-check
+        run: |
+          HELM_CRDS_PATH="./helm-charts-cloned/charts/atlas-operator-crds/templates"
+
+          filesToCopy=()
+          for filename in ./bundle/manifests/atlas.mongodb.com_*.yaml; do
+          absName="$(basename "$filename")"
+          echo "Verifying file: ${filename}"
+          if ! diff "$filename" ${HELM_CRDS_PATH}/"$absName"; then
+          filesToCopy+=("$filename")
+          fi
+          done
+
+          fLen=${#filesToCopy[@]}
+          if [ "$fLen" -eq 0 ]; then
+            echo "No CRD changes detected"
+            exit 0
+          fi
+
+          echo "The following CRD changes detected:"
+          for (( i=0; i < fLen; i++ )); do
+            echo "${filesToCopy[$i]}"
+          done
+
+          for (( i=0; i < fLen; i++ )); do
+            echo "Copying ${filesToCopy[$i]} to ${HELM_CRDS_PATH}/"
+            cp "${filesToCopy[$i]}" ${HELM_CRDS_PATH}/
+          done
+
+      - name: Verify if RBAC were changed
+        id: rbac-check
+        run: |
+          HELM_RBAC_FILE=../helm-charts-cloned/charts/atlas-operator/rbac.yaml
+
+          yq '.spec.install.spec.clusterPermissions[0].rules' ./bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml > rbac.yaml
+
+          echo "Comparing RBAC for CSV to RBAC in AKO helm chart"
+          if ! diff rbac.yaml "$HELM_RBAC_FILE"; then
+            echo "Copying RBAC"
+            cp rbac.yaml "$HELM_RBAC_FILE"
+          else
+            echo "No changes detected"
+          fi
```

Comment:
Why not make this into an script (eg. `scripts/update-helm.sh`) and call it though `devbox` and `make`. Eg:
```
  run: devbox run --  'make update-helm'
```

That way all scripts dependencies will be using the environment set by our locked devbox dependencies. Also we are able to run and test this locally.

Reply:
Precisely because it is a lot of bash I'd rather have it in a regular bash a script file, not even in make. It is easier to test and we can invoke outside of the CI as needed.

Whether it is one or 2 scripts, I don't have a strong opinion about that. The make file invocation can do both in one go: `run: devbox run -- 'make update-crds update-rbac'`

Reply:
I reverted it back to two bash scripts included into Makefile